faq.md

```diff
@@ -5,64 +5,150 @@ layout: faq
 1. TOC
 {:toc}
 
-# What is this website?
+# Overview Questions
 
-This website is a collection of tutorials developed and maintained by the worldwide Galaxy community. You can learn more about this effort with reading our [article](https://www.biorxiv.org/content/early/2018/04/05/225680).
+## What is this website?
 
-# How can I get some help?
+This website is a collection of tutorials developed and maintained by the [worldwide Galaxy community](https://galaxyproject.org/). You can learn more about this effort by reading our [article](https://www.biorxiv.org/content/early/2018/04/05/225680).
```

Comment:
Something like:"a collection of hands-on tutorials that are designed to be interactive and are built around Galaxy."

Reply:
Can we get rid of the OSX advertisement ;)
As a more serious comment I'm not sure we should recommend this split screen. I'm worried about people that do not have such big screens around. Or is this standard nowadays?